SokolTextGame/World.cs

```diff
@@ -11,11 +11,13 @@ private void CommandDictionary()
             commands["who"] = new WhoAmI();
             commands["look"] = new Look();
             commands["where"] = new WhereAmI();
+            commands["go"] = new GoTo();
```

Comment:
Please change the name of the `GoTo` command to just `Go`. The command names have to be generic as they should be theoretically able to process several user inputs, not just the `go to location` in this case.